### DISCUSSION TITLE
Add `plot` method for `default_priors`

### DISCUSSION THREAD

**Discussion by GidonFrischkorn** [venpopov/bmm#191] [Ideas]
When I was thinking about the `default_priors` for the `m3`, I thought having an easy way of plotting the `default_priors` on the parameters and native scale would be nice. I was thinking about something similar to the plots you generated for the "extract info" vignette. What do you think about this?

**Comment by venpopov** [comment]
You read my mind. I was thinking about the same thing earlier this week. In prncile you can do this by running the model with "sample_prior = only" but that requires compiling the model, extracting the draws, transforming them, etc. So it would be great I you want to make such a function

**Comment by venpopov** [comment]
![Screenshot_20240328_121802](https://github.com/venpopov/bmm/assets/12783882/9093db3c-a9eb-4990-afe3-5fa240fd7cf3)
You can convert the discussion to an issue by a button below
